quickstart.md

```diff
@@ -270,12 +270,284 @@ In case the user wants to run the workload on a TDX capable hardware, using QEMU
+
+For the SEV encrypted image use case, it is required to host the key used to encrypt the container image from the `simple-kbs`. At launch time, the kata shim will retrieve the container encryption key from the `simple-kbs` by providing a conditional proof that fulfills a policy configured on the `simple-kbs`. A common key release policy is to store the guest firmware measurement and require that the kata shim provides a matching value during launch.
+
+Currently, the SEV unencrypted image use case also requires the `simple-kbs` to be deployed. This will change in a future CoCo release.
+
+The SEV `simple-kbs` implementation allows for a policy to be set that verifies the guest launch measurement before releasing the secret used to encrypt the container image.
+
+`docker-compose` is required to run the `simple-kbs` and its database in docker containers:
+
+* Debian Ubuntu:
+
+  ```
+  sudo apt install docker-compose
+  ```
+
+Clone the repository for specified tag:
+```
+simple_kbs_tag="0.1.1"
+git clone https://github.com/confidential-containers/simple-kbs.git
+(cd simple-kbs && git checkout -b "branch_${simple_kbs_tag}" "${simple_kbs_tag}")
+```
+
+Run the service with `docker-compose`:
+```
+(cd simple-kbs && sudo docker-compose up -d)
+```
+
+#### Calculate the SEV Guest Measurement and Store the simple-kbs Policy
+
+This step describes the process to calculate the SEV guest firmware measurement and create a key release policy in the `simple-kbs` requiring the kata shim to provide a matching measurement.
+
+`pip` is required to install the `sev-snp-measure` utility. `mysql-client` is required to insert the key release policy into the `simple-kbs` database. `jq` is required to json parse responses on the command line.
+
+* Debian Ubuntu:
+
+  ```
+  sudo apt install -y python3-pip mysql-client jq
+  ```
+
+[sev-snp-measure](https://github.com/IBM/sev-snp-measure) is a utility used to calculate the SEV guest firmware measurement with provided ovmf, initrd, kernel and kernel append input parameters. Install it using the following command:
+
+```
+sudo pip install sev-snp-measure
+```
+
+The path to the guest binaries required for measurement is specified in the kata configuration. Set them:
+
+```
+ovmf_path="/opt/confidential-containers/share/ovmf/OVMF.fd"
+kernel_path="/opt/confidential-containers/share/kata-containers/vmlinuz-sev.container"
+initrd_path="/opt/confidential-containers/share/kata-containers/kata-containers-initrd.img"
+```
+
+The kernel append line parameters are included in the SEV guest firmware measurement. Provided here is a common value for these parameters:
+
+```
+append="tsc=reliable no_timer_check rcupdate.rcu_expedited=1 \
+i8042.direct=1 i8042.dumbkbd=1 i8042.nopnp=1 i8042.noaux=1 noreplace-smp reboot=k \
+cryptomgr.notests net.ifnames=0 pci=lastbus=0 console=hvc0 console=hvc1 quiet panic=1 \
+nr_cpus=1 scsi_mod.scan=none agent.config_file=/etc/agent-config.toml"
+```
```

Comment:
I would probably just direct people the script below from the start rather than giving them the default and then telling them what to do if it fails. It seems like a pain to keep this default up to date when kata changes.